.circleci/config.yml

```diff
@@ -1,7 +1,7 @@
 version: 2.1
 
 orbs:
-  hmpps: ministryofjustice/hmpps@3.12
+  hmpps: ministryofjustice/hmpps@5
```

Comment:
I thought about doing this too - decided it's a bit non-determinstic 🤷 Though it should be fine!